scripts/build_qat_envoy.sh

```diff
@@ -0,0 +1,203 @@
+#!/usr/bin/env bash
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set -e
+
+if [ -f /etc/os-release ]; then
+	source /etc/os-release
+else
+	source /usr/lib/os-release
+fi
+
+readonly YQ_BIN="$(realpath ${PWD}/yq)"
```

Comment:
Shellcheck reports a bunch of problems with this file, for example `${PWD}` lacking quotes. Could you run the command and see which ones could be fixed? You can annotate false positives.

Reply:
oops!, I forgot to enable shellcheck in emacs, I'll fix all errors